SECURITY.md

```diff
@@ -6,8 +6,8 @@ These versions of `@checkdigit/typescript-config` are currently being supported
 
 | Version | Supported          |
 | ------- | ------------------ |
-| \>= 3.x | :white_check_mark: |
-| \< 3.0  | :x:                |
+| \>= 5.x | :white_check_mark: |
+| \< 4.0  | :x:                |
```

Comment:
not sure if it should be `< 5.0` ? or should the contents in `SECURITY.md` stay the same?

Reply:
no it should be 5.0